mlflow/keras.py

```diff
@@ -111,9 +111,16 @@ def log_model(keras_model, artifact_path, conda_env=None, **kwargs):
 def _load_model(model_file):
     import keras.models
     import h5py
-    # NOTE: Keras 2.2.3 does not work with unicode paths in python2. Pass in h5py.File instead of
-    # string to avoid issues.
-    with h5py.File(os.path.abspath(model_file), "r") as model_file:
+
+    from distutils.version import StrictVersion
+
+    if StrictVersion(keras.__version__) >= StrictVersion("2.2.3"):
+        # NOTE: Keras 2.2.3 does not work with unicode paths in python2. Pass in h5py.File instead
+        # of string to avoid issues.
+        with h5py.File(os.path.abspath(model_file), "r") as model_file:
+            return keras.models.load_model(model_file)
+    else:
+        # NOTE: Older versions of Keras only handle filepath.
```

Comment:
Why was this changed? Is it meant to be in this PR? It's OK if this is a bug fix but kind of weird

Reply:
Yeah it's a bugfix. The current mlflow version does not work with older Keras versions. I thought it was too small a fix to make into a separate PR but I can extract it out.

Reply:
It's fine as long as it was intended, but it's also okay to have very small PRs.